### DISCUSSION TITLE
Font Awsome icons are not showing in Activeadmin resource for rails 7 project

### DISCUSSION THREAD

**Discussion by huzaifabutt** [activeadmin/activeadmin#8302] [Help]
Hey everyone,
I have been using font awesome for quite some time in various projects. I recently made a new project in rails 7 and im making an Admin panel in it, (using activeadmin gem)
I added 'font-awesome-sass' gem in my application to use different icons...but for some reason i cant access any of these icons in my active admin resources. However, if i use it outside of admin in normal PostsController...they work perfectly fine....
what am i doing wrong here?

**Comment by tagliala** [comment]
Hi,

I guess that this configuration is not using webpack or other js/node based asset pipelines, but I think that the problem is that FA's css is not being loaded by activeadmin

After you run the generator as per docs at https://activeadmin.info/0-installation.html, you should find `app/assets/stylesheets/active_admin.scss`. There you should import Font Awesome's scss according to the asset pipeline that your application is using

Feel free to share a demo application on GitHub, as a member of both FA and ActiveAdmin teams I will be glad to take a look

**Comment by huzaifabutt** [comment]
It worked. I downloaded fontawsome zip locally and placed the in app/assets/fonts folder
and then i imported them as  @import "fontawesome/css/all.min.css";

**Comment by tagliala** [comment]
> I downloaded fontawsome zip locally and placed the in app/assets/fonts folder
and then i imported them as https://github.com/import "fontawesome/css/all.min.css";

I'm glad it works, but I do not suggest to bundling the library, like bloating the repo with vendor files, updates, asset caching and fingerprinting.

I see that `font-awesome-sass` gem still depends on `sassc`, which is not ideal in these days.

There is a PR FortAwesome/font-awesome-sass#229 to support other sass compilers (missing cssbundling-rails), but if you for some reason use yarn in the asset pipeline or you have other npm , I would suggest to load the library through `package.json`
